compiler/natives/src/embed/internal/embedtest/embed_test.go

```diff
@@ -0,0 +1,12 @@
+//go:build js
+// +build js
+
+package embedtest
+
+import (
+	"testing"
+)
+
+func TestUninitialized(t *testing.T) {
+	t.Skip("reflect.DeepEqual error for empty slice and variadic")
```

Comment:
```suggestion
	t.Skip("reflect.DeepEqual error for empty slice and variadic: https://github.com/gopherjs/gopherjs/issues/1147")
```

Reply:
Actually, I sent https://github.com/gopherjs/gopherjs/pull/1150 to address the issue with reflect.DeepEqual(). As soon at it is merged, you should be able to re-enable this test case.

Reply:
fixed